templates/admin/streams/index.html.twig

```diff
@@ -0,0 +1,65 @@
+{% extends 'admin/admin.html.twig' %}
+{% trans_default_domain 'admin' %}
+
+{% block content %}
+	<h2>
+		{{ 'streams.index.title'|trans }}
+		<span class="right">
+				<a href="{{ path('admin_streams_new') }}" class="btn green">
+					{{ 'default.action.add'|trans }}
+					<i class="fa fa-plus" aria-hidden="true"></i>
+				</a>
+			</span>
+	</h2>
```

Comment:
Maybe have something a little more like this to avoid having the h2 working as a container

```suggestion
  <div>
	<h2>{{ 'streams.index.title'|trans }}</h2>
	<span class="right">
		<a href="{{ path('admin_streams_new') }}" class="btn green">
			{{ 'default.action.add'|trans }}
			<i class="fa fa-plus" aria-hidden="true"></i>
		</a>
	</span>
  </div>
```